### PR TITLE
Add build configuration while publishing packages

### PR DESCRIPTION
**Pull request by qinezh**

### CODE REVIEW

**Comment by qinezh** [comment]
#719

File: build.ps1 [hunk 1]
```diff
@@ -67,11 +67,21 @@ Write-Host "Start to build project"
 foreach ($folder in (dir "src"))
 {
     if (Test-Path (Join-Path $folder.FullName "project.json")) {
```
Comment (hellosnow) [inline build.ps1]: > { [](start = 62, length = 2)

move { to next line?

File: build.ps1 [hunk 2]
```diff
@@ -67,11 +67,21 @@ Write-Host "Start to build project"
...
-        & dotnet publish $folder.FullName -o target\$configuration\$folder
+        & dotnet build $folder.FullName -c $configuration -f net452
         if ($lastexitcode -ne 0) { Write-Error "dotnet build $folder error, exit code: $lastexitcode"; Pop-Location }
     }
 }
```
Comment (hellosnow) [inline build.ps1]: extract common function?

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**